active-record/src/main/java/com/iluwatar/active/record/MagicWand.java

```diff
@@ -0,0 +1,264 @@
+package com.iluwatar.active.record;
+
+import org.h2.tools.DeleteDbFiles;
+
+import java.sql.*;
+
+/**
+ * "Every single wand is unique and will depend for its character on the particular tree and magical creature
+ * from which it derives its materials. Moreover, each wand, from the moment it finds its ideal owner, will begin
+ * to learn from and teach its human partner." @ Garrick Ollivander
+ * <p/>
+ * Created by Stephen Lazarionok.
+ */
+public class MagicWand {
+
+    private Long id;
+
+    private Double lengthInches;
+
+    private WandWoodType wood;
+
+    private WandCoreType core;
+
+    public Long getId() {
+        return id;
+    }
+
+    public Double getLengthInches() {
+        return lengthInches;
+    }
+
+    public WandWoodType getWood() {
+        return wood;
+    }
+
+    public WandCoreType getCore() {
+        return core;
+    }
+
+    public void setLengthInches(Double lengthInches) {
+        this.lengthInches = lengthInches;
+    }
+
+    public void setWood(WandWoodType wood) {
+        this.wood = wood;
+    }
+
+    public void setCore(WandCoreType core) {
+        this.core = core;
+    }
+
+    @Override
+    public String toString() {
+        final StringBuilder sb = new StringBuilder("MagicWand(");
+        sb.append(this.getWood());
+        sb.append(", ");
+        sb.append(this.getCore());
+        sb.append(", ");
+        sb.append(this.getLengthInches());
+        sb.append(" inch(es))");
+        return sb.toString();
+    }
+
+    /**
+     * ***********************************
+     * ****** Data access methods
+     * ***********************************
+     */
+
+    private static final String SELECT_SQL = "select * from wand where id = ?";
+    private static final String DELETE_SQL = "delete from wand where id = ?";
+    private static final String UPDATE_SQL = "update wand set length_inches = ?, wood = ?, core = ? where id = ?";
+    private static final String CREATE_SQL = "insert into wand values(?, ?, ?, ?)";
+
+    static {
```

Comment:
@stephen-lazarionok Yes I hear you. I am also willing to keep the example simple. We have introduced consequences section in our pattern description with #205 . And I feel discussing these points will help us in the learning process and we can get something out of it for consequences section as well. Discussions on design patterns tend to get opinionated but hey that's the fun part. Ain't it? As far as the implementation is concerned I am happy with it.